framework/src/application.ts

```diff
@@ -279,7 +279,12 @@ export class Application {
 				endpoints: this._rootEndpoints(),
 				events: [APP_EVENT_READY.replace('app_', ''), APP_EVENT_SHUTDOWN.replace('app_', '')],
 			});
-			await this._stateMachine.init(this.logger, this.config.genesis, this.config.modules);
+			await this._stateMachine.init(
+				this.logger,
+				this.config.genesis,
+				this.config.generator,
```

Comment:
generator config can be removed from state machine and  `init` interface for module

Reply:
removed